packages/dotcom-build-code-splitting/package.json

```diff
@@ -6,12 +6,11 @@
   "types": "src/index.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "tsc": "../../node_modules/.bin/tsc --incremental",
     "clean": "npm run clean:dist && npm run clean:node_modules",
     "clean:dist": "rm -rf dist",
     "clean:node_modules": "rm -rf node_modules",
-    "build": "npm run build:node",
-    "build:node": "npm run tsc -- --module commonjs --outDir ./dist/node",
+    "build:node": "tsc",
+    "build": "node run build:node",
```

Comment:
**issue**: I don't think this script should have changed – `node run` isn't a valid command and results in an error when you run `npm run build:node` as it tries to load a module called `run`. Looks like this is present in all the `package.json`'s in the `packages` subdirectory.

Reply:
Looks like my find and replace skills have failed me. Fixing it now!

Reply:
Fixed 👍